src/js/components/DataFilter/DataFilter.js

```diff
@@ -15,6 +15,7 @@ const minSelectSearchOptions = 10;
 const defaultRangeSteps = 20;
 
 const getValueAt = (valueObject, pathArg) => {
+  if (!pathArg) return undefined;
   if (valueObject === undefined) return undefined;
```

Comment:
```suggestion
  if (!pathArg || valueObject === undefined) return undefined;
```
Could we change to this so we only need one `return undefined` statement?